src/Workspaces/Core/Portable/NamingStyles/EditorConfig/EditorConfigNamingStyleParser_NamingRule.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.Diagnostics.Analyzers.NamingStyles
+{
+    internal static partial class EditorConfigNamingStyleParser
+    {
+        private static bool TryGetSerializableNamingRule(
+            string namingRuleTitle,
+            SymbolSpecification symbolSpec,
+            NamingStyle namingStyle,
+            IReadOnlyDictionary<string, object> conventionsDictionary,
+            out SerializableNamingRule serializableNamingRule)
+        {
+            var severity = GetRuleSeverity(namingRuleTitle, conventionsDictionary);
```

Comment:
If this returns default because it can't be found, we will create a rule with a hidden severity.  Should we instead not create a rule at all?